server/command/command.go

```diff
@@ -153,17 +153,19 @@ type command struct {
 	handler HandlerMap
 }
 
-func (c *command) help(_ ...string) (*model.CommandResponse, error) {
+func (c *command) help(_ ...string) (string, error) {
 	helpText := helpTextHeader + helpText
-	return c.postCommandResponse(helpText), nil
+	return helpText, nil
 }
 
 func (c *command) postCommandResponse(text string) *model.CommandResponse {
```

Comment:
Can we remove this methods as it's not longer used?

Reply:
### Sure